wallet.js

```diff
@@ -190,17 +195,22 @@ async function get_address_txs({ address }) {
 }
 
 async function fetch_most_recent_unconfirmed_send() {
+    const IGNORE_UTXOS_BELOW_SATS = get_min_sat_utxo_limit();
+
     if (WALLET_TYPE === 'core') {
         const recent_transactions = listtransactions()
-        const unconfirmed_sends = recent_transactions.filter(it => it.category === 'send' && it.confirmations === 0)
+        const unconfirmed_sends = recent_transactions.filter(
+            it => it.category === 'send' && it.confirmations === 0 && BTC_to_satoshi(it.amount) >= IGNORE_UTXOS_BELOW_SATS
```

Comment:
can we log if we are ignoring any small ones? (similar to how we do in line 229 of this file, or 218 of the old file)

Reply:
![image](https://github.com/deezy-inc/sat-hunter/assets/159653027/ef8331f3-4eb8-4468-95cc-457b52126a39)